sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -59,21 +58,23 @@ az cognitiveservices account create \
     --location westus2 \
     --yes
 ```
+
+## Authenticate the client
 In order to interact with the Text Analytics service, you will need to create an instance of the Text Analytics client,
 both the asynchronous and synchronous clients can be created by using `TextAnalyticsClientBuilder` invoking `buildClient()`
 creates a synchronous client while `buildAsyncClient()` creates its asynchronous counterpart.
 
 You will need an **endpoint** and either a **key** or **AAD TokenCredential** to instantiate a client object. 
 
-##### Looking up the endpoint
+### Looking up the endpoint
 You can find the **endpoint** for your Text Analytics resource in the [Azure Portal][azure_portal] under the "Keys and Endpoint",
 or [Azure CLI][azure_cli_endpoint].
 ```bash
 # Get the endpoint for the text analytics resource
 az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "endpoint"
 ```
 
-##### Create a Text Analytics client with key credential
+### Create a Text Analytics client with key credential
```

Comment:
I think the guidance was to update the header formatting only this line and L106

Reply:
Yes, correct @samvaity

Reply:
(1) rearrange `Authenticate the client` is neccessary. Because it was put at the wrong section before. https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/textanalytics/Azure.AI.TextAnalytics#authenticate-the-client

(2) Remove this section is neccessary. It is because of .NET does has it as well.

(3) `### Looking up the endpoint`. is because it is the child section in `## Authenticate the client`. And `## Authenticate the client` is the sibling of `## Get Start`

Reply:
@czubair  I think this change will more reasonable. Can you have it review again? If not, I will revert the changes.